docs/aws/audit/cloudfrontmonitoring/rules/cloudfront_custom_ssl_certificate_remediation.mdx

```diff
@@ -1,6 +1,233 @@
+</Accordion>
+
+</AccordionGroup>
+</Tab>
+<Tab title='Cause'>
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Sign in to the AWS Management Console and open the CloudFront console at https://console.aws.amazon.com/cloudfront/.
+
+2. In the CloudFront console, in the top navigation pane, click on "Distributions".
+
+3. In the "Distributions" panel, you will see a list of all your CloudFront distributions. Click on the ID of the distribution you want to check.
+
+4. In the distribution settings, click on the "General" tab. Look for the "SSL Certificate" section. If the "Custom SSL Certificate (example.com)" option is selected, it means a custom SSL certificate is set for the distribution. If the "Default CloudFront Certificate (*.cloudfront.net)" option is selected, it means a custom SSL certificate is not set.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. First, you need to install and configure AWS CLI on your local machine. You can do this by following the instructions provided by AWS. Make sure you have the necessary permissions to access the CloudFront distributions.
+
+2. Once the AWS CLI is set up, you can list all the CloudFront distributions using the following command:
+
+   ```
+   aws cloudfront list-distributions --query 'DistributionList.Items[*].{ID:Id,Domain:DomainName}'
+   ```
+   This command will return a list of all the CloudFront distributions along with their ID and domain name.
+
+3. To check the SSL certificate of a specific CloudFront distribution, you can use the following command:
+
+   ```
+   aws cloudfront get-distribution-config --id <distribution-id>
+   ```
+   Replace `<distribution-id>` with the ID of the distribution you want to check. This command will return the configuration of the specified CloudFront distribution.
+
+4. In the output of the above command, look for the `ViewerCertificate` field. If the `ACMCertificateArn` field is present and has a value, it means that a custom SSL certificate is set for the distribution. If the `ACMCertificateArn` field is not present or does not have a value, it means that a custom SSL certificate is not set for the distribution.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install and configure AWS SDK for Python (Boto3): Before you can start writing Python scripts to manage AWS resources, you need to set up authentication credentials for your AWS account. You can do this by installing AWS CLI and configuring it with your access key ID and secret access key.
+
+   ```bash
+   pip install awscli
+   aws configure
+   ```
+
+2. Import the necessary modules and create a session: In your Python script, you need to import the Boto3 module and create a session using your credentials.
+
+   ```python
+   import boto3
+   session = boto3.Session(
+       aws_access_key_id='YOUR_ACCESS_KEY',
+       aws_secret_access_key='YOUR_SECRET_KEY',
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>